packages/events/TopLevelEventTypes.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+export type TopLevelType = number | string;
```

Comment:
Can this be Flow opaque type? So that we don't rely on what it is.

Reply:
Yeah seems to work but it becomes a bit ugly because I have to force-cast (`((x: any): OpaqueType`) in the two places where I create `TopLevelType` out of thin air. 

Let me push the commit so you can see.